addons/website/static/tests/tours/snippet_popup_display_on_click.js

```diff
@@ -92,10 +95,17 @@ registerWebsitePreviewTour("snippet_popup_display_on_click", {
         run: "click",
     },
     ...clickOnSave(),
+    {
+        trigger: "body .o_notification_manager:not(.o_upload_progress_toast):empty:hidden",
+    },
     {
         content: "Wait content of iframe is loaded",
         trigger: ":iframe main:contains(enhance your)",
     },
+    {
+        content: "Wait form is patched",
+        trigger: ":iframe form#contactus_form input[name=company]:value(yourcompany)",
```

Comment:
@pipu-odoo This seems to be [failing without demo data](https://runbot.odoo.com/odoo/runbot.build.error/116103), could you take a look?
Thanks